cdap-master/src/main/java/co/cask/cdap/data/tools/DatasetUpgrader.java

```diff
@@ -53,13 +54,15 @@
   private final LocationFactory locationFactory;
   private final QueueAdmin queueAdmin;
   private final HBaseTableUtil hBaseTableUtil;
-  private final DatasetFramework namespacedFramework;
+  private final DatasetFramework dsFramework;
   private static final Pattern USER_TABLE_PREFIX = Pattern.compile("^cdap\\.user\\..*");
```

Comment:
Opened a JIRA for this and another thing I noticed while making these updates.
https://issues.cask.co/browse/CDAP-1761